prow/cmd/deck/runlocal

```diff
@@ -13,25 +13,72 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
could you please add the PR description here?

Reply:
Done